lib/reply.js

```diff
@@ -10,6 +10,8 @@ const flatstr = require('flatstr')
 const fastseries = require('fastseries')
 const runHooks = fastseries()
 
+const isError = Symbol.for('is-error')
```

Comment:
You should use just `Symbol` for performances reasons. Since every symbol is different you must declare it in one file and export it.
@mcollina thoughts?

Reply:
there are no performance reasons to do that, more keeping it private. I agree.